xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -482,4 +418,37 @@ require(['jquery'], function ($) {
     // Autofocus on permalink field
     modal.find('.form-control').focus();
   })
+  /**
+   * Delete: Events triggered after the delete modal is displayed
+   */
+  $(document).on('shown.bs.modal' , '#deleteModal', function (event) {
+    var modal = $(this);
+    var button = $(event.relatedTarget);
+    /**
+     * Ajax request made for deleting the comment.
+     * Delete the HTML element on succes (replace it with a small notification message). Display error message on failure.
+     */
+    modal.find('input.btn.btn-danger').click( function(event) {
+      $.ajax({
+        url : button.prop('href'),
+        success : function() {
+          var comment = button.parentsUntil(".reply");
```

Comment:
Not very important, but you could use `'` instead of `"`, at least for consistency.